### DISCUSSION TITLE
Timescale DB Tool kit for windows

### DISCUSSION THREAD

**Discussion by nagcse42** [timescale/timescaledb-toolkit#807] [Feature Discussion]
Hi Team

Is there any way to add timescale DB extension for windows? or any future plan to release timescale DB toolkit for windows?

**Comment by syvb** [comment]
Windows is not supported, and there are no plans to add Windows support. Consider using Windows Subsystem for Linux (WSL).

**Reply by nagcse42** [reply]
@syvb  do you have any documentation or wiki how it use Windows Subsystem for Linux (WSL). It's really helpful.
